spec/requests/api/vms_spec.rb

```diff
@@ -89,7 +89,7 @@ def update_raw_power_state(state, *vms)
   context "Vm software subcollection" do
     let(:sw1) { FactoryGirl.create(:guest_application, :vm_or_template_id => vm.id, :name => "Word")  }
     let(:sw2) { FactoryGirl.create(:guest_application, :vm_or_template_id => vm.id, :name => "Excel") }
-    let(:vm_software_url)      { "#{vms_url(vm.id)}/software"    }
```

Comment:
Although I approve of this change, it doesn't seem related to the rest of this PR